src/FakerDotNet/FakerContainer.cs

```diff
@@ -28,6 +29,7 @@ public FakerContainer()
             Number = new NumberFaker();
             Random = new RandomFaker();
             Time = new TimeFaker();
+            Space = new SpaceFaker();
```

Comment:
You'll need to change this to `Space = new SpaceFaker(this)`, so that you can inject the `IFakerContainer` into the constructor.